internal/ias/internal/oidc/oidc-client.go

```diff
@@ -0,0 +1,86 @@
+package oidc
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+//go:generate mockery --name=Client --outpkg=mocks --case=underscore
```

Comment:
should we delete the line as this is really not a mock file?

Reply:
This line can be used to generate the mocks from the IDE. I put it there instead of the makefile, since I saw you also had it like this in one of your teams projects: https://github.com/kyma-project/nats-manager/blob/26766c5ac781ce0d2a9b0a875bc87fec8e60df57/pkg/manager/nats.go#L19